docs/topics/spiders.rst

```diff
@@ -259,6 +259,29 @@ to give data more structure you can use :class:`~scrapy.Item` objects::
             for href in response.xpath('//a/@href').getall():
                 yield scrapy.Request(response.urljoin(href), self.parse)
 
+.. _spiderargs:
+
+
+    .. method:: update_settings()
+        
+        The ``update_settings`` method allows you to modify the settings of a spider. It is a classmethod that is called to 
+        override the configuration of your spider. When called, it receives the spider's settings and passes ``custom_settings``
+        or an empty dictionary to the ``setdict`` method to register the setting with a priority of spider. It is recommended to
+        use ``update_settings`` rather than ``custom_settings`` when creating class hierarchies for spiders. This is because the parent's
+        ``custom_settings`` may be shadowed, but the parent's ``update_settings`` method can still be called.
```

Comment:
This is somewhat the right syntax to address this, but this documentation should go higher up in this document, before the “Let's see an example::” line above. I would put it between from_crawler and start_requests, since I assume that it when it is called; if it is called before from_crawler, then I would put it before.

Also, please consider using `.. automethod::`` instead of `.. method::`, and moving this documentation to the docstring of the actual method in code.